asv.conf.json

```diff
@@ -0,0 +1,184 @@
+{
+    // The version of the config file format.  Do not change, unless
```

Comment:
I take it ASV supports comments in JSON? Because typically that is not allowed.

Reply:
The comments are from the template generated using `asv`. I thought it would be useful to keep them. Should I remove it then?

Reply:
Addressed this in: https://github.com/tardis-sn/tardis/pull/2260/commits/0ed52ccdcad46decd14f5f66f590507ace744687

Would open another pull request for docs soon, to make people know about the config file